src/Simplex/Chat.hs

```diff
@@ -2210,7 +2215,8 @@ acceptFileReceive user@User {userId} RcvFileTransfer {fileId, xftpRcvFile, fileI
   case (xftpRcvFile, fileConnReq) of
     -- direct file protocol
     (Nothing, Just connReq) -> do
-      connIds <- joinAgentConnectionAsync user True connReq . directMessage $ XFileAcpt fName
+      dm <- directMessage $ XFileAcpt fName
+      connIds <- joinAgentConnectionAsync user True connReq dm
```

Comment:
```suggestion
      connIds <- joinAgentConnectionAsync user True connReq <=< directMessage $ XFileAcpt fName
```

Reply:
a lot of noice because of not having the version in scope – why not just make it available in the scope of directMessage?

Or s/./<=</ - just changing one composition with another

Reply:
or

```
connIds <- joinAgentConnectionAsync user True connReq =<< directMessage (XFileAcpt fName)

Reply:
by introducing an intermediate variable in all these places it only becomes harder to read, not easier.